app/views/mailers/notify_mailer/trusted_role_email.html.erb

```diff
@@ -2,17 +2,33 @@
   Hey <%= @user.name %>!
 </p>
 <p>
-  We are happy to let you know that your <%= community_name %> account has just been upgraded to "trusted" status.
+  We are happy to let you know that you've been granted "trusted" status on <%= community_name %>.
 </p>
 <p>
-  This gives you the power to privately downvote content which is low quality or spam and harassment, alongside a bit of additional functionality to help ensure the success of the community. Anywhere you now see a shield on the site indicates a place you can go to join our community's crowd-sourced moderation initiative.
+  This update allows you to make use of a few extra features — each of which furthers our goal of cultivating a welcoming and constructive community.
 </p>
 <p>
-  You may occasionally receive notifications about optional moderation actions, but you can  <a href="<%= app_url(user_settings_path(:notifications)) %>">unsubscribe</a> if you do not want to receive these notices.
+  Here's an overview of your new abilities:
+  <ul>
+    <li>Use reactions (👎, 👍, 🤢) to moderate content</li>
+    <li>Quickly flag rule-breaking behavior to <%= community_name %> staff</li>
+    <li>Rate the experience level of a post</li>
+  </ul>
 </p>
 <p>
-  <b><a href="<%= app_url("/community-moderation") %>">Click here for full details about what you can do with the trusted role.</b></a>
+  <b>For details on all available features and how to access them, visit our <a href="<%= app_url("/community-moderation") %>"><%= community_name %> Trusted User Guide.</b></a>
```

Comment:
```suggestion
  <b>For details on all available features and how to access them, visit our <a href="<%= app_url("/community-moderation") %>"><%= community_name %> Trusted User Guide</a>.</b>
```
Since the `<a>` tag is inside the `<b>`, let's nest them in the same order to preserve HTML semantics here